### PR TITLE
Unknown / unsupported hash functions do not break

### PR DESCRIPTION
**Pull request by devd**
This wording makes that clear

### CODE REVIEW

File: specs/subresourceintegrity/spec.markdown [hunk 1]
```diff
@@ -15,7 +15,7 @@ Sites and applications on the web are rarely composed of resources from
 only a single origin. For example, authors pull scripts and styles from a
 wide variety of services and content delivery networks, and must trust
 that the delivered representation is, in fact, what they expected to
-load. If an attacker can trick a user into downloading content from
+load. If an attacker can trvick a user into downloading content from
```
Comment (fmarier) [inline specs/subresourceintegrity/spec.markdown]: typo :)

File: specs/subresourceintegrity/spec.markdown [hunk 2]
```diff
@@ -277,10 +277,11 @@ When a hash function is determined to be insecure, user agents MUST deprecate
...
-Validation using unsupported hash functions always fails (see the "[Does
-resource match metadataList][match]" algorithm below). Authors are therefore
-encouraged to use strong hash functions, and to begin migrating to stronger
-hash functions as they become available.
+To allow authors to switch to stronger hash functions without being held back by older
+user agents, validation using unsupported hash functions acts like no integrity value 
+was provided (see the "[Does resource match metadataList][match]" algorithm below). 
+Authors  are encouraged to use strong hash functions, and to begin migrating to 
+stronger hash functions as they become available.
 </section><!-- /Framework::Cryptographic hash functions::Agility -->
```
Comment (fmarier) [inline specs/subresourceintegrity/spec.markdown]: I like this.

**Comment by devd** [comment]
whoops .. sorry! fixed.

**Comment by fmarier** [comment]
I'll give @metromoxie and @mozfreddyb a day or two to jump in before merging, but it looks good to me and is definitely better than what we've got at the moment.

**Comment by mozfreddyb** [comment]
:+1: :+1:

**Comment by metromoxie** [comment]
lgtm

**Pull request merged**